src/System.Management.Automation/engine/parser/Compiler.cs

```diff
@@ -1146,17 +1160,42 @@ private static Attribute NewCmdletBindingAttribute(AttributeAst ast)
             return result;
         }
 
-        private static Attribute NewParameterAttribute(AttributeAst ast)
+        private static Attribute NewExperimentalAttribute(AttributeAst ast)
         {
-            CheckNoPositionalArgs(ast);
+            int positionalArgCount = ast.PositionalArguments.Count;
+            if (positionalArgCount != 2)
+            {
+                throw InterpreterError.NewInterpreterException(null, typeof(MethodException), ast.Extent,
+                    "MethodCountCouldNotFindBest", ExtendedTypeSystem.MethodArgumentCountException, ".ctor",
```

Comment:
Is it worth turning `".ctor"` into a constant?

Reply:
I found 8 instances of this string, so maybe yes. But it's better to be done in a separate PR.